clustering/src/main/java/io/hyperfoil/controller/Data.java

```diff
@@ -31,6 +32,8 @@ final class Data {
    final Map<String, StatisticsSnapshot> perAgent = new HashMap<>();
    final Map<String, IntObjectMap<StatisticsSnapshot>> lastStats = new HashMap<>();
    final List<StatisticsSummary> series = new ArrayList<>();
+   final Map<String, List<Histogram>> intervalHistogramsPerAgent = new HashMap<>();
+   final List<Histogram> intervalHistograms = new ArrayList<>();
```

Comment:
Had to track the histograms at `Data` level, still nor sure whether this could introduce problems 🤔

Reply:
2 things:
- it would be great to not collect anything unless required i.e. if the hdr output is configured
- it would be good to pre-size the list based on the duration

The last is a minor TBH since I won't expect total duration to exceed 15 minutes or so, which means 15 * 60 histograms.